tests/infra_test.go

```diff
@@ -1106,6 +1106,20 @@ var _ = Describe("[Serial][owner:@sig-compute]Infrastructure", func() {
 				Expect(found).To(Equal(false), "Node can't contain label "+v1.CPUModelLabel+obsoleteModel)
 			})
 
+			It("should update node with new cpu model vendor label", func() {
```

Comment:
@ksimon1 from what I saw so far in the functional tests code, we don't return in a middle of `It` as a sign for a successful test, although technically it may be right. Also I didn't notice common usage of `Fail`. I think I can suggest you to use the following approach, it will also save you from iterating the node resources:
```
selector := metav1.ListOptions{LabelSelector: v1.CPUModelVendorLabel}
nodes, err := virtClient.CoreV1().Nodes().List(context.Background(), selector)
Expect(err).ToNot(HaveOccurred())
Expect(len(nodes)).To(BeNumerically(">=", 1))
```

Reply:
@ksimon1 Another concern - how can we be sure that every cluster these tests would run on will have this label on the nodes that run virt-handler?

Reply:
Issue is, that the label is in format `cpu-vendor.node.kubevirt.io/<vendorName>` and AFAIK the label selector doesn't allow regex. 
To your second question: I added check which controls if there are any node with kvm, if not, it skips whole labeller section.